learn/wavemaker-release-notes/v11-5-0.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "WaveMaker 11.5.0 - Release date: 04 December 2023"
+id: "v11-5-0"
+sidebar_label: "v11.5.0 - latest"
+---
+
+WaveMaker has released version 11.5.0 with new features including Appium Support, Dockerfiles in a project, and enhancements like Angular framework support for Prefab workspace, NG studio upgrade to version 15, and other bug fixes.
+
+## New Features
+
+### Dockerfiles in Project
+
+Introduced Dockerfiles in the project to build and deploy applications. This helps the user to use an appropriate Dockerfile for a specific project which stores all the prerequisite versions that are needed for the application.
+
+### Appium Support - React Native
+
+Appium is a popular mobile UI test automation framework, and it supports various programming languages to write test cases. In WaveMaker, the component generates a unique identifier that is based on the widget name and attaches it as a testId property on all React Native elements that it uses.
+
+### Docker Image for Local Web Preview - React Native
+
+Introduced **wm-rn-web-preview** Docker image to allow users to generate the local web preview of the application. Docker image is pre-configured with essential libraries which lets the user update to the latest version of the development environment by pulling the latest docker image. This simplifies the process of building and running Wavemaker applications in the local environment. 
```

Comment:
error  Incorrect usage of the term: “Wavemaker”, use “WaveMaker” instead  terminology